packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -27,7 +27,10 @@ const convertRemoteParticipantsToParticipantListParticipants = (
       remoteParticipants
         // temporarily hiding lobby participants in ACS clients till we can admit users through ACS clients
         .filter((participant: RemoteParticipantState) => {
-          return participant.state !== 'InLobby';
+          return (
```

Comment:
update the comment here to reflect the new behavior
```suggestion
        // Hide lobby participants in ACS clients till we can admit users through ACS clients.
        // Hide Participants in the idle or connecting stages also, unless they are a phoneNumber in which case connecting reflects on outgoing call
        .filter((participant: RemoteParticipantState) => {
          return (
```